Core/FileSystems/DirectoryFileSystem.cpp

```diff
@@ -900,7 +900,7 @@ std::vector<PSPFileInfo> DirectoryFileSystem::GetDirListing(std::string path) {
 		localtime_r((time_t*)&s.st_atime,&entry.atime);
 		localtime_r((time_t*)&s.st_ctime,&entry.ctime);
 		localtime_r((time_t*)&s.st_mtime,&entry.mtime);
-		if (!listingRoot || (wcscmp(findData.cFileName, L"..") && wcscmp(findData.cFileName, L".")))
+		if (!listingRoot || (wcscmp(dirp->d_name, L"..") && wcscmp(dirp->d_name, L".")))
```

Comment:
`dirp->d_name` is `char` array, no matching function for call to `wcscmp`

Reply:
Derp.  That's what I get for copy/pasting.  I changed it afterward (I had a big if around the whole thing), and decided this change was better, then copy/pasted it both places.

-[Unknown]